### PR TITLE
Update Hello World tutorial - Closes #797

### LINKED ISSUE #797: Update Hello world tutorial
Update the Hello World tutorial to support SDK v5: https://github.com/LiskHQ/lisk-sdk-examples/pull/77

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #797

### CODE REVIEW

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 1]
```diff
@@ -1,637 +1,727 @@
...
+│   ├── hello_plugin <3>
+│   │   └── index.js
+│   ├── index.js
+│   └── package.json
+....
 
-=== Supported platforms
+<1> A custom module, which is containing the on-chain logic related to the `hello` property .
+<2> A custom asset, which is handling the logic for the hello transaction inside the `hello_module`.
+<3> A custom module, which is providing additional API endpoints related to the `hello_module`.
```
Comment (nazarhussain) [inline modules/ROOT/pages/tutorials/hello-world.adoc]:
```suggestion
<3> A plugin, which is providing additional off-chain API endpoints related to the `hello_module`.
```

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 2]
```diff
@@ -1,637 +1,727 @@
...
+		process.exit(1);
+	});
+----
 
-[[sendable]]
-=== How to create sendable transaction objects
+=== The hello module
 
-When you are in the root folder of the Hello World application, create the file that will hold a code snippet to create the transaction object:
+The hello module extents like every other module from the `Basemodule`.
```
Comment (nazarhussain) [inline modules/ROOT/pages/tutorials/hello-world.adoc]: Not sure to use `extents` or `extends`.

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 3]
```diff
@@ -1,637 +1,727 @@
...
+<2> The name for the asset.
+Must be unique within the module.
+<3> The asset ID.
+Must be unique within the module.
+<4> `schema` defines structure for the data in the transaction asset.
+<5> `validate` validates the data in the transaction asset, before it is applied.
+<6> `apply` applies the desired changes in the database, based on the data in the transaction asset.
 
-Now that we know how to <<sendable,create sendable transaction objects>>, let's discover the application through a simple react frontend.
+=== The hello API plugin
```
Comment (nazarhussain) [inline modules/ROOT/pages/tutorials/hello-world.adoc]:
```suggestion
=== The Hello API plugin
```

**Review by nazarhussain — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/tutorials/hello-world.adoc (hunk 2 above)
Comment (Tschakki) [reply modules/ROOT/pages/tutorials/hello-world.adoc]:
```suggestion
The hello module extends like every other module from the `BaseModule`.
```

**Review by nazarhussain — APPROVED** [review]

**Pull request merged**